packages/cbioportal-ts-api-client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "cbioportal-ts-api-client",
   "description": "cBioPortal API Client for TypeScript",
-  "version": "1.0.0-beta.0",
+  "version": "1.1.0-beta.0",
```

Comment:
this should be `1.0.0-beta.1`

Reply:
why btw should this be `.1` and not `1.1.0`?